index.html

```diff
@@ -1,10 +1,29 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>Personal Website - About Me</title>
+  	<link rel="stylesheet" type="text/css"
+      href="https://fonts.googleapis.com/css?family=Open+Sans">
+    <link href="https://fonts.googleapis.com/css?family=PT+Serif" rel="stylesheet">
+    <title>Jett House - About Me</title>
     <link rel="stylesheet" type="text/css" href="css/style.css">
   </head>
   <body>
-    
+  	<ul class="nav">
+  		<li><a href="index.html">About Me</a></li>
+  		<li><a href="resume.html">Resume</a></li>
+  	</ul>
+  	<div class="content">
+	  	<h1>About Me</h1>
+	  	<div>
+      		<img src="images/profile.jpg">
+      	</div>
+	  	<h2>Hi there, I'm Jett.</h2>
+      	<h3>Let me tell you a bit about myself.</h3>
+	  	<h4>The Early Years</h4>
+	  	<p>NYC born 'n' raised... actually on second thought, let's cut to the chase.</p>
+	  	<h4>The Chase</h4>
+	  	<p>By day I'm a product designer, by night I'm a front-end coding student. I excel at synergizing solutions from multiple stakeholders through a rigorous process of colaborative optimization. My prefered mode of transport is hot air balloon (I rarely take the subway anymore).</p>
+	  	<h5>If you're curious about where I've left my mark, continue on to my <a href="resume.html">resume</a>.</h5>
```

Comment:
Way to add in these extras. It sucks to have to nav all the way back up and this is a natural jumping point.